oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/LastRevRecoveryAgent.java

```diff
@@ -309,6 +317,8 @@ public void sweepUpdate(Map<Path, UpdateOp> updates)
         long startOfScan = clock.getTime();
         long lastLog = startOfScan;
 
+        final List<Revision> pseudoBcRevs = new ArrayList<Revision>();
```

Comment:
Can use diamond operator.
```suggestion
        final List<Revision> pseudoBcRevs = new ArrayList<>();
```